lib/error_tracker/integrations/oban.ex

```diff
@@ -32,35 +32,31 @@ defmodule ErrorTracker.Integrations.Oban do
 
   # https://hexdocs.pm/oban/Oban.Telemetry.html
   @events [
-    [:oban, :job, :start],
     [:oban, :job, :exception]
   ]
 
   @doc false
   def attach do
-    if Application.spec(:oban) do
+    if ErrorTracker.Integrations.Utils.application_spec(:oban) do
       :telemetry.attach_many(__MODULE__, @events, &__MODULE__.handle_event/4, :no_config)
     end
   end
 
-  @doc false
-  def handle_event([:oban, :job, :start], _measurements, metadata, :no_config) do
```

Comment:
I don't think we should remove this. It works for developers that report exceptions from their workers calling to `ErrorTracker.report/2` or `ErrorTracker.report/3`.

If we remove this part those errors wouldn't have this context information anymore.

Reply:
I did not think about this use case I will put back the start event